Wikipedia/UI-V5/WMFLocationManager.m

```diff
@@ -6,8 +6,6 @@
 
 NS_ASSUME_NONNULL_BEGIN
 
-static CLLocationDistance WMFMinimumDistanceBeforeUpdatingLocation = 1.0; //meters before we update location
-
 @interface WMFLocationManager ()<CLLocationManagerDelegate>
```

Comment:
So looking at the history of this file I saw one commit about the dealloc method… so that seems good.

But the rest seems to be unexplained in the commit messages and mostly looks to be futzing around with the logic…  

However, I can clearly see that the behavior has been changed. 

Did you fix a bug? What was the bug? What's better now? Did you make tradeoffs?

Hesitant to merge changes in this file without explanations on how the behavior was changed and why.